docs/azure-setup.md

```diff
@@ -79,7 +79,7 @@ An ARM template can be opened within Azure Portal via the following REST API:
 
 The URI can be one for either an Azure Blob or a GitHub file.  For example,
 
-    https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Ftongtest.blob.core.windows.net%2Fcntk%2FazureDeployMainTemplate.json
+    https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmmlspark%2Fmaster%2Ftools%2Fdeployment%2FazureDeployMainTemplate.json
```

Comment:
So the concern here is the file can be relocated so that the url is broken? This is the doc that describes the usage of those deployment templates and scripts. In this particular example, the url is what Azure API requires. (All Azure ARM templates are hosted in GitHub and referenced using GitHub url in the published documents.) Don't see a way to avoid it. At other places in this doc, the relative path is used for linkage. If a file is relocated, the relative path still needs to be updated. 

The other usage of my blob storage (tongtest) is to host the prebuilt hdfs-mount and cntk. We can move them to https://mmlspark.azureedge.net I assume. This looks like a different problem than this one though.